.rubocop.yml

```diff
@@ -1,13 +1,18 @@
-inherit_gem:
-  bixby: bixby_default.yml
+inherit_from:
+  - .bixby.yml
 
-Metrics/BlockLength:
+RSpec/DescribeClass:
   Exclude:
-    - spec/deploy_spec.rb
+    - smoke_spec/deploy_spec.rb
 
-RSpec/DescribeClass:
+Metrics/BlockLength:
+  Max: 38
   Exclude:
-    - spec/deploy_spec.rb
+    - 'spec/**/*.rb'
+    - 'smoke_spec/**/*.rb'
 
 RSpec/ExampleLength:
-  Enabled: false
+  Exclude:
```

Comment:
Out of curiosity: why the change from disabling RSpec/ExampleLength to excluding all the specs from it?  I think both wind up doing the same thing.

Reply:
the `Enabled: false` was still triggering on the smoke specs for some reason. this fixed it so I didn't spend more time trying to figure out why.